projects/demo/src/modules/components/input-date-multi/input-date-multi.module.ts

```diff
@@ -0,0 +1,33 @@
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {ReactiveFormsModule} from '@angular/forms';
+import {TuiAddonDocModule, tuiGetDocModules} from '@taiga-ui/addon-doc';
+import {
+    TuiDropdownModule,
+    TuiHintModule,
+    TuiLinkModule,
+    TuiTextfieldControllerModule,
+} from '@taiga-ui/core';
+import {TuiInputDateMultiModule} from '@taiga-ui/kit';
+
+import {InheritedDocumentationModule} from '../abstract/inherited-documentation/inherited-documentation.module';
+import {TuiInputDateMultiExample1} from './examples/1';
+import {ExampleTuiInputDateMultiComponent} from './input-date-multi.component';
+
+@NgModule({
+    imports: [
+        CommonModule,
+        TuiLinkModule,
+        TuiHintModule,
+        TuiDropdownModule,
+        TuiAddonDocModule,
+        ReactiveFormsModule,
+        TuiInputDateMultiModule,
+        TuiTextfieldControllerModule,
+        InheritedDocumentationModule,
```

Comment:
should we import `TuiMobileCalendarDialogModule` ?

Reply:
fixed